russian/call_friend/call_friend.md

```diff
@@ -0,0 +1,5 @@
+ Видя стену из зефира вы звоните своему корешу с вопросом чем он вас таким вчера угощал. Вам рекомендуют выпить чегото бодрящего а потом прогулятся и все пройдет. И вы не замедляя 
+
+[Идёте варить кофе.](awaken/stand-up/coffee/coffee.md) 
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.